.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+image: ci4rail/gitpod-esp
+tasks:
+  - name: shell env
+    before: |
+      echo 'source $IDF_PATH/export.sh' >~/.bashrc.d/100-esp-idf
+      echo 'source $IDF_PATH/export.sh' >>~/.zshrc
+      mkdir ~/.ssh
+      echo -e "${SSH_PRIVATE_KEY//_/\\n}" > ~/.ssh/id_rsa
+      chmod 600 ~/.ssh/id_rsa
+  - name: pre-commit install
+    before: |
+      echo 'PIP_USER="no"' >~/.bashrc.d/110-fix-pip-user
+      export PIP_USER="no" && pre-commit install && pre-commit
```

Comment:
We should find a way to install pre-commit hooks also in components. We could add a script in gitpod-esp image and call it from this .gitpod.yml